html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../home-page/home-page.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
```

Comment:
You don't need `../about-me` here. You are already in the `about-me` folder, so you can just do `about-me.html`